share/spice/bikesharing/phlindego/bikesharing_phlindego.js

```diff
@@ -0,0 +1,77 @@
+(function(env) {
```

Comment:
the path's for this files should be `/spice/bike_sharing/phlindego/bike_sharing_phlndego.js`